api/src/services/stardust/influx/influxDbClient.ts

```diff
@@ -0,0 +1,292 @@
+import { INanoDate, InfluxDB, IPingStats, IResults, toNanoDate } from "influx";
+import moment from "moment";
+import { INetwork } from "../../../models/db/INetwork";
+import {
+    IAddressesWithBalanceDailyInflux, IAliasActivityDailyInflux, IAvgAddressesPerMilestoneDailyInflux,
+    IBlocksDailyInflux, ILedgerSizeDailyInflux, INftActivityDailyInflux, IOutputsDailyInflux,
+    IStorageDepositDailyInflux, ITimedEntry, ITokensHeldPerOutputDailyInflux, ITokensHeldWithUnlockConditionDailyInflux,
+    ITokensTransferredDailyInflux, ITransactionsDailyInflux, IUnclaimedGenesisOutputsDailyInflux,
+    IUnclaimedTokensDailyInflux, IUnlockConditionsPerTypeDailyInflux
+} from "../../../models/influx/influxData";
+import { CACHE_INIT, InfluxDbClientCache } from "../../../models/influx/influxDbCache";
+import {
+    ADDRESSES_WITH_BALANCE_DAILY_PARAMETERIZED_QUERY, ALIAS_ACTIVITY_DAILY_PARAMETERIZED_QUERY,
+    AVG_ACTIVE_ADDRESSES_PER_MILESTONE_DAILY_PARAMETERIZED_QUERY, BLOCK_DAILY_PARAMETERIZED_QUERY,
+    LEDGER_SIZE_DAILY_PARAMETERIZED_QUERY, NFT_ACTIVITY_DAILY_PARAMETERIZED_QUERY, OUTPUTS_DAILY_PARAMETERIZED_QUERY,
+    STORAGE_DEPOSIT_DAILY_PARAMETERIZED_QUERY,
+    TOKENS_HELD_BY_OUTPUTS_DAILY_PARAMETERIZED_QUERY, TOKENS_HELD_WITH_UC_DAILY_PARAMETERIZED_QUERY,
+    TOKENS_TRANSFERRED_DAILY_PARAMETERIZED_QUERY, TRANSACTION_DAILY_PARAMETERIZED_QUERY,
+    UNCLAIMED_GENESIS_OUTPUTS_DAILY_PARAMETERIZED_QUERY, UNCLAIMED_TOKENS_DAILY_PARAMETERIZED_QUERY,
+    UNLOCK_CONDITIONS_PER_TYPE_DAILY_PARAMETERIZED_QUERY
+} from "./influxQueries";
+
+/**
+ * The collect job interval (1h).
+ */
+const COLLECT_DATA_FREQ_MS = 1000 * 60 * 60;
+
+const NANOSECONDS_IN_MILLISECOND = 1000000;
+
+/**
+ * The timestamp to collect data FROM for Shimmer.
+ * Tuesday, 27 September 2022 00:00:00
+ */
+const DEFAULT_FROM_TIMESTAMP_MS = 1664229600000;
```

Comment:
if statistics also is used on eg testnet, timestamp shouldn't be a constant because starting point of network is different :thinking:

Reply:
Indeed. We'll remove the need for it.